bcda/_site/404.html

```diff
@@ -8,20 +8,20 @@
 	<!-- Metadata for Twitter -->
 	<meta name="twitter:title" content="Your awesome title" />
 	<meta name="twitter:description" content="Write an awesome description for your new site here. You can edit this line in _config.yml. It will appear in your document head meta (for Google search resu..." />
-	<meta name="twitter:image" content="/assets/img/default-share-image.jpg" />
+	<meta name="twitter:image" content="http://localhost:4000/assets/img/default-share-image.jpg" />
```

Comment:
I think it's correct to switch these from relative to absolute URLs, but shouldn't they use the domain of the site?

Reply:
_site is supposed to be ignored.  It gets built during the deploy process.  I was running it locally (not even in our app)  and it put in the full path.  When I rebuilt it with the docker container it reverted to a relative path which I think is actually correct.

Reply:
FWIW, I don't think anyone will ever see this `404` page unless we configure the go `FileServer` to respond with it in the event of a `404`.

Reply:
Good point, NBD as it's not being used at the moment. But I think in general these Twitter and Open Graph URLs are expected to be absolute. They are in the examples here: https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started

I believe Twitter has a tool for checking how the meta tags will be read, so maybe we want to do that when the site is public.